spec/develop.js

```diff
@@ -56,13 +56,15 @@ describe('development server', function () {
     })
     .then(function (body) {
       assert(body.length);
+      assert(body.indexOf('<!doctype html>') === -1);
```

Comment:
How about asserting in the previous `.then` that the `Content-Type` response header is `application/javascript` instead of `text/html` as well?